Assets/MRTK/SDK/Features/UX/Scripts/PulseShader/SurfacePulse.cs

```diff
@@ -8,20 +8,21 @@
 using UnityEngine;
 using UnityEngine.Serialization;
 
-namespace Microsoft.MixedReality.Toolkit.Experimental.SurfacePulse
+namespace Microsoft.MixedReality.Toolkit.UI.SurfacePulse
 {
     /// <summary>
-    /// Script for generating pulse shader effect on the surface.
+    /// Script for generating pulse shader effect on the surface of a mesh.  
     /// </summary>
     [AddComponentMenu("Scripts/MRTK/SDK/SurfacePulse")]
     public class SurfacePulse : MonoBehaviour, IMixedRealityPointerHandler
```

Comment:
I'm also a little confused by the pattern in `Start`:

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/e6beaf9b7794a335e2fc18d23546084357e3b95a/Assets/MRTK/SDK/Features/UX/Scripts/PulseShader/SurfacePulse.cs#L215-L217

Why do we need to add an additional script on the surface mapping? Shouldn't this script be applied to the mapping parent? Where is this script supposed to be applied that the `OnPointerClicked` event is received, if not the spatial mapping itself?

Reply:
Discussed with @keveleigh offline. #9179 for tracking.